.github/workflows/pull-request-checks.yaml

```diff
@@ -4,6 +4,44 @@ on:
     branches: [ develop ]
 
 jobs:
+  check-clang-format:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
```

Comment:
To get this to work you’d want `fetch_depth: 0` here to check out the entire repository rather than create a shallow clone (the default).